src/component/plasmic/shared/PlasmicActionBar.module.css

```diff
@@ -28,13 +29,14 @@
   padding-bottom: 0px;
 }
 .root__isActive {
-  max-height: 431px;
+  max-height: 1000px;
```

Comment:
Unfortunately the animation when clicking on the ActionBar won't work without a `max-height`. We could use JS to set this dynamically based on content, but this is easier and should cover most use cases.